content/docs/05.developer-guide/13.cicd/index.md

```diff
@@ -4,77 +4,220 @@ title: Create a CI/CD pipeline
 
 ## Introduction
 
-In Kestra, the number of flows you develop can grow quickly, and maintaining them can get challenging.
-CI/CD pipelines are a great way to automate your flows' validation and deployment.
-This section will guide you through creating a CI/CD pipeline for your Kestra flows.
+Manual deployments can be challenging. CI/CD pipelines are a great way to automate the validation and deployment of your workflows. This section explains several ways of creating a CI/CD pipeline for your Kestra resources.
 
 ## Why a CI/CD pipeline?
 
-CI/CD pipelines provide an easy and reliable way to ensure that your flows are always deployed as soon as they're done.
-It integrates nicely with VCS (Version Control System) like Git and can easily integrate into your development workflow.
+A CI/CD process helps ensure fast and reliable deployments. Your changes get deployed automatically, as soon as they get peer-reviewed and merged to a Git branch.
 
-## How to create a CI/CD pipeline?
+## CI/CD for data workflows
+
+There are several ways to create a CI/CD pipeline for your flows in Kestra. Pick one of the options listed below that best suits your needs.
 
-There are several ways to create a CI/CD pipeline. At Kestra, we propose three different methods:
 
 <ChildTableOfContents :max="1" />
 
-## Helpers
+### Kestra CLI 
 
-Kestra has some _helpers_ function that can help during flow development; they allow separating a flow description on multiple files thanks to an include function, see [Helpers](./04.helpers.md).
+Kestra CLI provides several [commands](./04.helpers.md) for validating and deploying your flows.
 
-## Useful CLI commands
+If you run Kestra in a container, you can access those as follows:
```

Comment:
I would prefere to only show the CLI command, the way people access the CLI (inside docker or not) is not relevant.
In the future, we will have a proper CLI separate from the Docker container used to run Kestra itself.